src/base/Modal/ReactPortal.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useLayoutEffect } from 'react'
+import { createPortal } from 'react-dom'
+
+function createWrapperAndAppendToBody(wrapperId) {
```

Comment:
to keep files clean, I'd suggest to create file util.js in this folder and move it there

Reply:
ok, got it. So I was thinking I'm writing code only in features while base, utils, share dirs are empty😅